setup.py

```diff
@@ -426,6 +426,17 @@ def make_deb():
     os.mkdir("./xpra-html5")
     shutil.copytree("./packaging/debian", "./xpra-html5/DEBIAN")
     install_html5("./xpra-html5/usr/share/xpra/www/", "uglifyjs")
+    # Move and symlink default-settings.txt
+    os.makedirs("./xpra-html5/etc/xpra/html5-client", exist_ok=True)
+    default_settings_files = { "default-settings.txt", "default-settings.txt.gz", "default-settings.txt.br" }
+    for filename in default_settings_files:
+        print("Checking existence of %s" % os.path.join("./xpra-html5/usr/share/xpra/www", filename))
+        if os.path.exists(os.path.join("./xpra-html5/usr/share/xpra/www", filename)):
+           os.rename(os.path.join("./xpra-html5/usr/share/xpra/www", filename),
```

Comment:
Instead of repeating the same path 4 times, please use a variable to make the code more readable. ie: `www_dir`.

Reply:
This will be fixed too